src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -5621,7 +5621,7 @@ private static string DescribeLoop(RegexNode node, RegexMethod rm)
 
         private static string ToHexStringNoDashes(byte[] bytes) =>
 #if NET
-            Convert.ToHexString(bytes);
+            Convert.ToHexString(bytes);kk
```

Comment:
Temp change for CI

Reply:
Assuming you just had this here to prove it was building in CI.   Please revert before checking in.